src/test/java/ca/corefacility/bioinformatics/irida/ria/unit/web/projects/ProjectSamplesControllerTest.java

```diff
@@ -306,7 +309,7 @@ public void testGetAjaxProjectSampleModels() {
 		));
 
 		when(sampleService
-				.getFilteredSamplesForProjects(any(List.class), any(List.class), any(String.class), any(String.class), any(String.class), any(Date.class), any(Date.class),
```

Comment:
Same question as for another test above about why are we now checking for `isNull()` instead of `any(String.class)`. For this test why are we also checking for `isNull() in place of these `any(Date.class)` `any(Date.class)`

Reply:
Same as above comment:

Mockito 1.x which we were on would accept null values for any() even if a class is expected.